src/index.css

```diff
@@ -0,0 +1,109 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  background-color: aliceblue;
+}
+
+header {
+  margin: 10px;
+}
+
+.flex {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+h1 {
+  font-size: 50px;
+}
+
+h2 {
+  font-size: 30px;
+}
+
+.container {
+  width: 100%;
+}
+
+.container .players {
+  width: 50%;
+  height: 300px;
+  justify-content: flex-start;
+  flex-direction: column;
+  padding: 0 40px;
+}
+
+.container .players .playersHeader {
+  display: flex;
+  width: 100%;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 0;
+  margin-top: 10px;
+}
+
+.container .players .playersHeader #btnRefresh {
```

Comment:
[OPTIONAL] It is not recommended to select the 'id' in a multiline selection. Instead, it is advisable to select it separately or through the class name for better organization and ease of management.

Reply:
Ohh Okay great, thank you very much